### PR TITLE
ignore network errors according to --load-error-handling param

### PR DESCRIPTION
**Pull request by quentindemetz**
piggy-backing on https://github.com/wkhtmltopdf/wkhtmltopdf/pull/4461

I am trying to generate PDFs of user-submitted HTML documents, some of these try to load invalid fonts.

As of today, using `--load-error-handling ignore` still leads to `Exit with code 1 due to network error: UnknownContentError` -> that's not what I expected.

I've taken into account the suggestion on #4461 and taken the opportunity to remove an unreachable line of code and its comment. Let me know if that's OK. 🦍

### CODE REVIEW

File: src/lib/multipageloader.cc [hunk 1]
```diff
@@ -435,16 +435,15 @@ void ResourceObject::amfinished(QNetworkReply * reply) {
...
-			// XXX: Notify network errors as higher priority than HTTP errors.
-			//      QT's QNetworkReply::NetworkError enum uses values overlapping
-			//      HTTP status codes, so adding 1000 to QT's codes will avoid
-			//      confusion. Also a network error at this point will probably mean
-			//      no HTTP access at all, so we want network errors to be reported
-			//      with a higher priority than HTTP ones.
-			//      See: http://doc-snapshot.qt-project.org/4.8/qnetworkreply.html#NetworkError-enum
-			error(QString("Failed to load %1, with network status code %2 and http status code %3 - %4")
-				.arg(reply->url().toString()).arg(networkStatus).arg(httpStatus).arg(reply->errorString()));
-			httpErrorCode = networkStatus > 0 ? (networkStatus + 1000) : httpStatus;
```
Comment (ashkulz) [inline src/lib/multipageloader.cc:447]: Why did you remove this?

Comment (quentindemetz) [reply src/lib/multipageloader.cc:447]: I couldn't find where the `httpErrorCode` variable was used later / I think it's unused.

Comment (quentindemetz) [reply src/lib/multipageloader.cc:447]: If the error message includes both the `networkStatus` and `httpStatus` variables, I'm not sure this logic is really necessary any more.
If you consider this point valid I can also amend lines 452-459 in the same way (show `networkStatus` and `httpStatus` in the error / warning message)

Comment (theonegri) [reply src/lib/multipageloader.cc:447]: @ashkulz thoughts on this?

Comment (theonegri) [reply src/lib/multipageloader.cc:447]: @ashkulz ping

Comment (theonegri) [reply src/lib/multipageloader.cc:447]: re-pinging @ashkulz

Comment (quentindemetz) [reply src/lib/multipageloader.cc:447]: pong
